### PR TITLE
Add AY22/23 Welcome Tea 

### PR DESCRIPTION
**Pull request by shenyih0ng**
- [ ] Add welcome tea physical signup link
- [ ] Confirm venue
- [ ] Food/pizza?

### CODE REVIEW

File: scripts/gen_fh.py [hunk 1]
```diff
@@ -8,12 +8,11 @@
 # As for numbering, it will take the next number
 # (e.g. if the previous post is FH #1000, the generated one will be FH #1001)
 # Please first update data/friday_hacks.yml before running this
+import re
 import yaml
-from datetime import datetime, timedelta
-from os import listdir
-from os.path import isfile, join
```
Comment (DrWala) [inline scripts/gen_fh.py:14]: Will removing this cause any issues?

File: content/post/2022-08-12-friday-hacks-223.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+#### Talk Description:
+
+Welcome back to Friday Hacks 👋
+
+We will be sharing about what we do and the importance of fostering hacker culture in NUS. We will also give an overview of the events that we have lined up for the upcoming academic year and how you can get involved in helping us spread the joy of hacking!
+
+All freshmen and current students from all faculties, as well as anyone else who is interested, are all welcome to attend!
+
+If you are unable to come join us physically, you are welcome to join us [online](https://nus-sg.zoom.us/j/89065336025?pwd=a0d4ZkRQMHZwNUFEL2hMcFFwZUNVdz09)!
```
Comment (DrWala) [inline content/post/2022-08-12-friday-hacks-223.md:26]: Is this link confirmed?

File: content/post/2022-08-12-friday-hacks-223.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
+---
+title: "Friday Hacks #223, August 12: Welcome Tea"
+date: 2022-08-03 22:34:26.523646
+author: Shen Yi Hong
+url: /2022/08/friday-hacks-223
+aliases:
+  - /welcometea2022
+nofh: true
```
Comment (DrWala) [inline content/post/2022-08-12-friday-hacks-223.md:8]: Is `nofh` supposed to be true?

**Review by DrWala — COMMENTED** [review]
Just some quick questions

File: content/post/2022-08-12-friday-hacks-223.md (hunk 3 above)
Comment (DrWala) [reply content/post/2022-08-12-friday-hacks-223.md:8]: ok I guess this is a generated file so it doesn't matter

File: scripts/gen_fh.py (hunk 1 above)
Comment (shenyih0ng) [reply scripts/gen_fh.py:14]: this chunk is actually part of a reordering of imports its not removed

File: content/post/2022-08-12-friday-hacks-223.md (hunk 2 above)
Comment (shenyih0ng) [reply content/post/2022-08-12-friday-hacks-223.md:26]: i think we can just use this? its hosted on my account

Comment (DrWala) [reply content/post/2022-08-12-friday-hacks-223.md:26]: Ok!

File: content/post/2022-08-12-friday-hacks-223.md (hunk 3 above)
Comment (DrWala) [reply content/post/2022-08-12-friday-hacks-223.md:8]: TIL, thanks!

File: scripts/gen_fh.py (hunk 1 above)
Comment (DrWala) [reply scripts/gen_fh.py:14]: Ok!

**Review by DrWala — APPROVED** [review]

**Pull request merged**